packages/manager/modules/pci/src/projects/project/storages/databases/database/databases/create-database/translations/Messages_fr_FR.json

```diff
@@ -5,5 +5,6 @@
   "pci_databases_databases_create_database_description": "Ajouter une base de données à votre service.",
   "pci_databases_databases_create_database_label": "Nom de la base de données:",
   "pci_databases_databases_create_database_success_message": "Votre nouvelle base de données {{database.name}} a été ajoutée",
-  "pci_databases_databases_create_database_error_message": "Une erreur est survenue lors de la création de votre base de données: {{message}}"
+  "pci_databases_databases_create_database_error_message": "Une erreur est survenue lors de la création de votre base de données: {{message}}",
+  "pci_databases_databases_create_database_banner": "Attention, le nom doit respecter les conditions suivantes : <br/>- Minimum {{min}} caractère<br/>- Maximum {{max}} caractères<br/>- Doit être composé uniquement de chiffres et de lettres."
```

Comment:
**nitpick:** We tend to avoid having markups in translation file (here: `<br/>`).
Could you please split this translation into multiple chunk?

Reply:
Fixed: https://github.com/ovh/manager/pull/5620/commits/36a994f957d25cc6adef8315f7e344b098d4286a